controller/WebController.php

```diff
@@ -588,10 +587,11 @@ public function getChangeList($request, $prop, $offset=0, $limit=200)
     }
 
     /**
-    * Formats the list of concepts as labels arranged by modification month
-    * @param Array $changeList
-    * @param string $lang the language for displaying dates in the change list
-    */
+     * Formats the list of concepts as labels arranged by modification month
+     * @param Array $changeList
```

Comment:
It seems like there are PHP types stated in both uppercase and lowercase (starting with) in PHPDocs all over the code base (array, for instance, here).

Well, I guess they do not matter. 😅

Reply:
I noticed that too. I **think** in this case I opened another code in this function, that returned `Array`, and then I just typed `Array` too :smile: but normally I use `string`, `boolean`, `array`…

Reply:
I guess it is OK at this point.

We should anyways add type hints/declarations directly into function arguments and return values as requested in #727 but that can be done en masse, simultaneously double checking these PHPDocs.